### PR TITLE
[#173130630] Upload Documents

### PR DESCRIPTION
**Pull request by NeilvB**
### Description of change

Give assessors document upload functionality. Permits PDF, jpeg and png files.

The biggest opportunity for code improvement in this PR is around the `DrawingController` actions. At the moment, this is mixed in with the actions for archiving a document. I suggest we tackle this as a separate refactoring task to create separate controllers for the two workflows.

### Story Link

https://www.pivotaltracker.com/n/projects/2441805/stories/173130630

### UI

Screenshot 2020-08-03 at 17 19 10

Screenshot 2020-08-03 at 17 19 21

Screenshot 2020-08-03 at 17 19 33

Screenshot 2020-08-03 at 17 19 39

### CODE REVIEW

File: app/policies/planning_application_policy.rb [hunk 1]
```diff
@@ -7,6 +7,26 @@ class PlanningApplicationPolicy < ApplicationPolicy
...
+  end
+
+  def update?
+    super || signed_in_editor?
+  end
+
+  def confirm_new?
+    signed_in_editor?
+  end
+
```
Comment (EGiataganas) [inline app/policies/planning_application_policy.rb:29]: you can replace these lines with
```ruby
alias_method :confirm_new?, :editor?
```

File: config/locales/en.yml [hunk 2]
```diff
@@ -9,3 +9,13 @@ en:
...
   dimensions: "Revise dimensions"
   other: "Other"
+
+  activerecord:
+    errors:
+      models:
+        drawing:
+          attributes:
+            plan:
+              unsupported_file_type: "Plan is not a supported file format"
```
Comment (Rhian) [inline config/locales/en.yml:19]: Should we add text to say which formats are permitted?

File: app/policies/planning_application_policy.rb (hunk 1 above)
Comment (NeilvB) [reply app/policies/planning_application_policy.rb]: Updated in https://github.com/unboxed/bops/pull/107/commits/fbe89c4d938e9250e78c9658809d4c03aa344484

File: config/locales/en.yml (hunk 2 above)
Comment (NeilvB) [reply config/locales/en.yml]: Updated in https://github.com/unboxed/bops/pull/107/commits/25a590ebb7e53196b51a0fe8668ed735499a4df8

**Pull request merged**